### DISCUSSION TITLE
serve cloud functions

### DISCUSSION THREAD

**Discussion by itsnotrisky** [TriPSs/nx-extend#66] [Ideas]
After generating a cloud function project, in the `project.json` file there are only `lint`, `test`, `build`, and `deploy` targets. Can we add `serve` for development with support to watch changes?

**Comment by TriPSs** [comment]
That is because there is a separate runner `nx g @nx-extend/gcp-functions:init-runner`, this will generate one project that you can use to serve all the functions.

**Reply by itsnotrisky** [reply]
Perfect, wasn't sure how it's all tied together.
